src/Microsoft.ApplicationInsights.AspNet/Extensibility/Implementation/Tracing/AspNetEventSource.cs

```diff
@@ -0,0 +1,121 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="AspNetEventSource.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+namespace Microsoft.ApplicationInsights.AspNet.Extensibility.Implementation.Tracing
+{
+    using System;
+    using System.Diagnostics.Tracing;
+
+    /// <summary>
+    /// Event source for Application Insights ASP.NET 5 SDK.
+    /// </summary>
+    [EventSource(Name = "Microsoft-ApplicationInsights-AspNet")]
+    internal sealed class AspNetEventSource : EventSource
```

Comment:
Similarly how we are doing for [Core](https://github.com/Microsoft/ApplicationInsights-dotnet/blob/1ccd5b466451db1884b242b6a58b3e9401a1fbcf/src/Core/Managed/Shared/Extensibility/Implementation/Tracing/CoreEventSource.cs), I think we want to differentiate each diagnostic message source with a separate event ID. This way we are ensuring message reuse and enable efficient filtering and search for diagnostic events.